app/models/namespace.rb

```diff
@@ -7,9 +7,10 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  team_id     :integer
-#  public      :boolean          default("0")
-#  registry_id :integer
+#  registry_id :integer          not null
 #  global      :boolean          default("0")
+#  description :text(65535)
+#  visibility  :integer
```

Comment:
Funnily enough, on my development environment nothing gets annotated on migration (and it used to happen), but if I call the rake task directly it will work.

*Update*: it looks like upgrading the `annotate` gem fixes my issue. I'll do this in a PR.

Reply:
it seems to be tricky keeping it up to date.